Testing/SystemTests/tests/framework/PowderReduceP2DTest.py

```diff
@@ -0,0 +1,126 @@
+import systemtesting
+from mantid.simpleapi import PowderReduceP2D
+
+
+class PowderReduceP2DTest(sytemtesting.MantidSystemTest):
+    def __init__(self):
+        systemtesting.MantidSystemTest.__init__(self)
+        self.tolerance = 1e-6
+        setUp()
+
+    def setUp(self):
+        self._sample = _sampleEventData()
```

Comment:
The method call is missing `self._...`
```suggestion
        self.sample =self. _sampleEventData()
```
Its the same for all of the ones below, I would also recommend removing the _ from the variable name, as some of these variable names clash with our method names

This needs to be done for all of them and also the call to `PowderReduceP2D` line 34.